packages/contracts/README.md

```diff
@@ -1,47 +1,42 @@
-# <h1 align="center"> Hardhat x Foundry Template </h1>
+# <h1 align="center"> Balancer Pool Metadata Contract</h1>
 
-**Template repository for getting started quickly with Hardhat and Foundry in one project**
+This package contains the source code for the Balancer Pool Metadata smart-contract, the [`PoolMetadataRegistry`](./src/PoolMetadataRegistry.sol).
 
-![Github Actions](https://github.com/devanonon/hardhat-foundry-template/workflows/test/badge.svg)
+# Getting started
 
-### Getting Started
+## Filling environment variables
 
-- Use Foundry:
+To be able to test and deploy the smart contract, you need to create a `.env` file in the `contracts` directory. Check the [`.env.example`](./.env.template) for information.
 
-```bash
-forge install
-forge test
-```
+## Install dependencies
 
-- Use Hardhat:
+If you did not clone the repository using the `--recursive-submodules` option, make sure to install the submodules with
 
 ```bash
-npm install
-pnpm hardhat test
+$ git submodule update --init --recursive   # in the balancer-pool-metadata directory
```

Comment:
`#question` does it make a difference if you run this outside of those directories?

```suggestion
$ git submodule update --init --recursive
```

Reply:
I tried to run it in the contract directory and it did not work